var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -83,12 +84,45 @@ class Proj(AutotoolsPackage):
     )
 
     # https://proj.org/install.html#build-requirements
-    depends_on("pkgconfig@0.9.0:", type="build", when="@6:")
     depends_on("googletest", when="@6:")
     depends_on("sqlite@3.11:", when="@6:")
     depends_on("libtiff@4.0:", when="@7:+tiff")
     depends_on("curl@7.29.0:", when="@7:+curl")
+    depends_on("pkgconfig@0.9.0:", type="build", when="@6: build_system=autotools")
+    depends_on("cmake@2.6.0:", type="build", when="build_system=cmake")
 
+    build_system("autotools", conditional("cmake", when="@5.0.0:"), default="cmake")
+
+
+class Setup:
+    def setup_run_environment(self, env):
```

Comment:
@alalazo can't remember, which of these belong in the package and which in the builder?

Reply:
If it helps the package audit check explicitly complained about the `setup_dependent_build_environment` method

Reply:
I think `setup_run_environment` in particular has to be in the package, not the builder. Not sure about the rest.

Reply:
I think `setup_dependent_run_environment` should also be in the package, not the build. @alalazo is there some documentation or code I can look at to get a full list of what belongs where?

Reply:
I've been working off of https://spack.readthedocs.io/en/latest/spack.html#spack.builder.Builder and whatever isn't there I'm putting in the package. Just missed the dependent version.